tests/CppWinRTComServerCustomModuleLockTests.cpp

```diff
@@ -0,0 +1,65 @@
+#include "pch.h"

```

Comment:
This file & corresponding test executable can be deleted if we de-templatize the notifier bype

Reply:
Changed the test to test `WIL_CPPWINRT_COM_SERVER_CUSTOM_MODULE_LOCK`, in case user need to further define custom type for the lock